bakery-src/scripts/gdocify_book.py

```diff
@@ -383,6 +383,17 @@ async def run_async():
     out_dir = Path(sys.argv[2]).resolve(strict=True)
     book_slugs_file = Path(sys.argv[3]).resolve(strict=True)
 
+    if len(sys.argv) >= 5:
+        worker_count = int(sys.argv[4])
+    else:  # pragma: no cover
+        try:
+            from multiprocessing import cpu_count
+            # Some container configurations prohibit this action
+            worker_count = cpu_count()
+        except Exception:
+            print("Error detecting cpu count, using 4 workers...")
+            worker_count = 4
```

Comment:
@TylerZeroMaster just curious why the worker count is set to 4. everything else looks good to me.

Reply:
@sparksam I am thinking that most processors today have at least four cores. Even if that is not true, I think performance can be gained from some level of multitasking because there is some setup and teardown time for each process that other tasks can be interleaved between.